WPMedia/fusion-news-theme-blocks
# THEMES-729 - Manual Promo Image Resizer Updates

## Description

Update small, medium, large and extra large manual promos to use resizer v2

Updated tests, and converted to RTL where needed

**Note**: Storybook showing changes for all manual promos due to image updated to use an image that is available via resizer v2

## Jira Ticket

- [THEMES-729](https://arcpublishing.atlassian.net/browse/THEMES-729)

## Test Steps

1. Checkout this branch `git checkout THEMES-729`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/small-manual-promo-block,@wpmedia/medium-manual-promo-block,@wpmedia/large-manual-promo-block,@wpmedia/extra-large-manual-promo-block`
3. Add each manual promo to a page and configure content - use the integrated photo search by using the image icon on the block preview to pick an image from Photo Center
4. Verify the image uses v2 resizer 

## Author Checklist

        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._
